main.py

```diff
@@ -580,7 +580,7 @@ def LoadDefaultSettings(self):
     app = QtGui.QApplication(sys.argv)
     app.setApplicationName("audio-visualizer")
     app.setOrganizationName("audio-visualizer")
-    window = uic.loadUi("mainwindow.ui")
+    window = uic.loadUi(os.path.join(os.path.dirname(os.path.realpath(__file__)), "mainwindow.ui"))
```

Comment:
This should maybe be stored in a variable so it can easily be used for other imported files, components, etc.